qcodes/instrument_drivers/Keysight/N52xx.py

```diff
@@ -344,6 +371,15 @@ def __init__(self,
                            get_parser=int,
                            set_cmd="CALC:PAR:MNUM {}",
                            vals=Numbers(min_value=1, max_value=24))
+
+        self.add_parameter(
+            "trigger_source",
+            get_cmd="TRIG:SOUR?",
+            set_cmd="TRIG:SOUR {}",
+            vals=Enum("EXT", "IMM", "INT", "MAN"),
+            set_parser=lambda value: "IMM" if value is "INT" else value
```

Comment:
can you add a note in comments on why this `set_parser`?

Reply:
Simply put, you want to select internal triggering, saying `trigger_source('IMM')` is unintuitive. I want to be able to do `trigger_source('INT')`